doc/model/axes/axis-cartesian-continuous.xml

```diff
@@ -382,6 +458,23 @@
             </c:documentation>
         </c:property>
 
+
+        <c:property name="ratio" type="number pvc.options.varia.TimeRatio" category="Continuous > Scale">
```

Comment:
See `preserveRatio`.
Also, despite the fact that probably the docs for `TimeRatio` already explain it detail, could we provide a simple string value example? Or else, maybe explain the ratio concept a bit better. Also, links to other real-time-related options would be helpful to illuminate on possible uses for this option.